docs/maps/import-geospatial-data.asciidoc

```diff
@@ -58,6 +58,54 @@ On the {kib} home page, you can upload a file and import it into an {es} index w
 . Use the file chooser to select a GeoJSON file.
 . Click *Import file*.
 
+[discrete]
+=== Draw features in a map
+
+Upload features into {es} by drawing lines, polygons, circles, bounding boxes, and points in a map.
+
+To create a new index for drawing:
+
+. <<maps-create, Create a map>>.
+. Click *Add layer*.
+. Select *Create index*.
+. Set *Index name*.
+. Click *Create index* button.
+
+To open an existing index for drawing:
+
+. <<maps-create, Create a map>>.
+
+. Click *Add layer*.
+
+. Select *Documents*.
+
+. Select the data view that points to your index. A <<data-views, data view>> can point to one or more indices. For feature editing, the data view must point to a single index.
+
+. Click *Add layer*.
+
+. Set *Scaling* to *Limit results to 10,000*.
+
+. In **Filtering**:
+** Clear the *Apply global search to layer data* checkbox.
+** If your data view contains a default time field, clear the *Apply global time to layer data* checkbox.
+
+. Click *Save & close*.
+
+. In the legend, click the layer name and select *Edit features*.
+
+When feature editing is open, a feature editing toolbox is displayed on the left side of the map.
+
+[role="screenshot"]
+image::maps/images/drawing_layer.png[]
+
+To draw features:
+
+. Click on the line, polygon, circle, bounding box, or point icon.
+. Move the mouse cursor over the map and follow the on screen instructions to draw a feature.
```

Comment:
To get the right formatting, add a line space after line 104 and put a "+" on that line.

Remove the two periods before "When"

It should look like this:

>.   Move the mouse cursor ....
>+
>When a feature is complete ....